### ISSUE TITLE
index.js 

### ISSUE THREAD

**Issue by PatriciaBethBarker** [PatriciaBethBarker/Final_Project#1]
I have an error I do not understand.  Can you help?

**Comment by thomaswilburn** [comment]
Maybe. What's the error?

**Comment by PatriciaBethBarker** [comment]
![index-error](https://cloud.githubusercontent.com/assets/7258182/8174016/c593b50c-138b-11e5-811f-1d4873d99588.png)

**Comment by thomaswilburn** [comment]
You have a mismatched punctuation--a missing brace or parentheses. In this case, it looks to me like you didn't correctly close the function call that starts on line 18.

**Comment by PatriciaBethBarker** [comment]
replaced models/blog.js with post.js because I was getting confused.  I dded moment in my post.js and this happened.

**Comment by thomaswilburn** [comment]
Okay, I don't think this is related to your models, though. It's an error after you added the hapi auth cookie plugin.

**Comment by PatriciaBethBarker** [comment]
![error-async](https://cloud.githubusercontent.com/assets/7258182/8174334/2dea4c52-1390-11e5-9392-2c528279b9ae.png)

**Comment by PatriciaBethBarker** [comment]
I have the async required int he db.js file.  Should I move that to the index.js?

**Comment by thomaswilburn** [comment]
You shouldn't move it, but you may need to add it. You need to require `async` into any module that uses it--just loading it in one file doesn't make it available in others.

**Comment by PatriciaBethBarker** [comment]
![asyncloaded](https://cloud.githubusercontent.com/assets/7258182/8185867/a89a9edc-13fb-11e5-9cf8-63de08a9f1b6.png)

**Comment by thomaswilburn** [comment]
Don't use the `-g` flag, that means you're listing global packages. `async` should be installed locally.

**Comment by PatriciaBethBarker** [comment]
grrrrrrrrrrrrrr, so do I have to uninstall ?

**Comment by thomaswilburn** [comment]
No, not at all. You can check to see if it's installed locally with `npm list --depth=0`, which will only show top-level packages in your directory.

**Comment by PatriciaBethBarker** [comment]
![list-of-npm](https://cloud.githubusercontent.com/assets/7258182/8188350/451a55c8-140a-11e5-8746-3e9ff05697b3.png)

**Comment by thomaswilburn** [comment]
Looks like you do need to install it locally. Don't forget to save it to your package: `npm install async --save`.

**Comment by PatriciaBethBarker** [comment]
OK   I removed the hapi-auth-cookie...and its looking for it now...

**Comment by thomaswilburn** [comment]
Why remove it? Extraneous means that it is installed, but not saved to your package. It doesn't mean you don't need it.

**Comment by PatriciaBethBarker** [comment]
![hatecookiesecret](https://cloud.githubusercontent.com/assets/7258182/8195138/167c1bea-1435-11e5-9298-7ca32b932a1d.png)

**Comment by PatriciaBethBarker** [comment]
it doesn't like my cookie secret.  I am really frstrrated....

**Comment by thomaswilburn** [comment]
No, it doesn't like the dash that's at the start of the line. That's why it has a little `^` pointing to it from beneath.

**Comment by PatriciaBethBarker** [comment]
Hates my code ;)
![hates-my-code](https://cloud.githubusercontent.com/assets/7258182/8196284/186929a2-143f-11e5-8402-c17a1f1b3760.png)

**Comment by thomaswilburn** [comment]
Well, it says "cannot add auth.scheme without a connection," so the logical answer is to move your `server.connection()` call up to above the `server.register` code.
